ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -102,6 +102,17 @@ chain:
         exit 0
       fi
       
+      # add cluster-info tags if exists
+      hosted_cluster_info_file=${SHARED_DIR}/hosted_cluster_info_file
+      if [[ -f "${hosted_cluster_info_file}" ]] ; then
+        while read -r TAG VALUE
+        do
+          printf 'Setting user tag to help cost usage analysis - %s: %s\n' "${TAG}" "${VALUE}"
+          COMMAND+=(--additional-tags="${TAG}=${VALUE}")
```

Comment:
This will append several `--additional-tags` options, is that expected ?

Reply:
yes, it is a slice param, we can set it multiple times